recipes/dd4hep/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "dd4hep" %}
+{% set version = "01-13" %}
+
+package:
+  name: {{ name }}
+  version: "{{ version.split('-')|map('int')|join('.') }}"
+
+source:
+  url: https://github.com/AIDASoft/DD4hep/archive/v{{ version }}.zip
+  sha256: 945a04cd04e9af23cdd4ba29a41bbff0fc388a66e5b640eb3f633596f130573d
+  folder: source
+
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
```

I guess if any packages ever depend on `dd4hep` they will likely be using it as a shared library? If so, it should use `run_exports`:
```suggestion
  skip: true  # [win]
  run_exports:
    - {{ pin_subpackage('dd4hep', max_pin='x.x') }}
```